backend/lib/endpoints/feedback.js

```diff
@@ -0,0 +1,36 @@
+const httpErrors = require("http-errors");
```

Comment:
please use the updated `httpErrors` that are included int he `fastify-sensible` package (see readme for more info, I haven't updated every endpoint yet because there are already some pull requests in progress)

Reply:
I just looked at that and saw that there is the option to use `reply` instead of throwing errors. What do you think about that? It looks a bit slimmer and still express what we need.

Reply:
I updated it to use fastify-sensible.

Reply:
There is always the option to use `reply` but it's not recommended to use it together with async/await.
If you read through the fastify docs about async/await, promises etc. and route handlers then you will see they recommend to use return and throw, without the `reply`, also because you could combine `return reply...` but it might get complicated.
see https://www.fastify.io/docs/latest/Routes/#async-await 
and https://www.fastify.io/docs/v2.6.x/Reply/#async-await-and-promises
and https://www.fastify.io/docs/latest/Routes/#promise-resolution
Also in fastify-sensible docs there are some examples, and to me it makes most sense to prefer only `return value` or `throw error` ... in this way we can also do proper error propagation, but I'm still trying to figure out the best systematic approach with fastify & async/await, suggestions welcome ofcourse if you think you can improve it.